jwst/datamodels/schemas/miri_mrsptcorr.schema.yaml

```diff
@@ -24,7 +24,7 @@ allOf:
       fits_hdu: DQ
       default: 0
       ndim: 2
-      datatype: int64
+      datatype: int32
```

Comment:
actually should be *u*int32 (unsigned)

Reply:
Hm, what's the best way to convert an existing array from regular int to uint?  Manually add an offset value and hack the BZERO in the header?  Passing it through np.uint looks like it goes to uint64 not uint32.

Reply:
Can you just do an `<my_array>.astype(np.uint32)`?

Reply:
Done